js/_website/src/lib/templates/gradio/03_components/navbar.svx

```diff
@@ -19,31 +19,6 @@
 gradio.Navbar(···)
 ```
 
-<!-- Embedded Component -->
```

Comment:
Let's merge to unbreak the website so we can release in the next day or two but is there any way to fix the underlying issue?